template/index.html

```diff
@@ -1,7 +1,191 @@
-{{ define "view" }}
+{{define "view"}}
 
-<div class="container my-2">
-    <h2>Index</h2>
+<div class="layout">
+  <div class="cards">
+
+    {{range .}}
+      <div class="card">
+        <div class="card-header-container">
+          <a href="" class="card-link">
+            <h2 class="title">Score #1</h2>
+          </a>
+        </div>
+        <div class="card-content-container">
+          <div class="histograms">
+            {{$numHistograms := (len .Histograms)}}
+            {{range .Histograms}}
+              <div class="score-histogram" style="width: calc(100% / {{$numHistograms}});">
+                <div class="top-player-name">{{.TopPlayerName}}</div>
+                <div class="host-table">
+                  {{$numStats := (len .Stats)}}
+                  {{range .Stats}}
+                    <div class="column-container" style="width: calc(100% / {{$numStats}});">
+                      <div class="column" style="height: {{.Value}}%;"></div>
+                    </div>
+                  {{end}}
+                </div>
+                <div class="histogram-name">{{.Name}}</div>
+              </div>
+            {{end}}
+          </div>
+        </div>
+        <div class="card-footer-container">
+          <a href="" class="card-link">
+            <span class="date">12 Sep 2020</span>
+          </a>
+        </div>
+      </div>
+    {{end}}
+
+  </div>
 </div>
 
-{{ end }}
+<style>
```

Comment:
css should be in separate file, but we need to configure static files serving (future develop)

Reply:
added issue #33